package.json

```diff
@@ -1,24 +1,23 @@
 {
-  "name": "d2l-intl",
-  "version": "2.1.0",
-  "description": "D2L internationalization APIs for number, date and time formatting and parsing.",
-  "main": "dist/index.js",
-  "types": "src/index.d.ts",
-  "module": "src/index.js",
+  "name": "@brightspace-ui/intl",
+  "version": "3.0.0",
+  "description": "Internationalization APIs for number, date, time and file size formatting and parsing in D2L Brightspace.",
+  "main": "lib/number.js",
```

Comment:
I didn't want to keep `index.js` around, since it would encourage folks to just import everything from there -- not just the things they needed.